src/dev/build/tasks/os_packages/package_scripts/post_trans.sh

```diff
@@ -4,5 +4,4 @@ if [ ! -f "${KBN_PATH_CONF}"/kibana.keystore ]; then
     /usr/share/kibana/bin/kibana-keystore create
     chown root:<%= group %> "${KBN_PATH_CONF}"/kibana.keystore
     chmod 660 "${KBN_PATH_CONF}"/kibana.keystore
-    md5sum "${KBN_PATH_CONF}"/kibana.keystore > "${KBN_PATH_CONF}"/.kibana.keystore.initial_md5sum
```

Comment:
This was in anticipation of a keystore versioning scenario that didn't materialize.  It's not used.